NEWS.md

```diff
@@ -1,4 +1,23 @@
-# pmtables (development version)
+# pmtables 0.7.0
+
+- New functionality to create table notes from glossary files (#326). 
+  - `read_glossary()` reads from `.tex` or `.yaml` formatted files, returning a
+    glossary object
+  - `select_glossary()` selects specific entries from a glossary object
+  - `as_glossary()` creates a glossary object on the fly or from a list
+  - `glossary_notes()` creates notes from a glossary object, a list, or the 
+    name of a glossary file
+  - `st_notes_glo()` creates and and attaches notes to a table in a pipeline
+- `stable_save_image()` added to create an image from a table and save to a 
+  specific location; this builds on `st_aspdf()` and `st_aspng()` and 
+  adding convenient syntax and options (#333). 
```

Comment:
The "and adding" reads a bit off to me.  Maybe "and `st_aspng()`, adding convenient [...]" or "and `st_aspng()` and adds convenient [...]"?

Reply:
Thanks; yes that didn't read right.